### ISSUE TITLE
Feat: Ability to enable logging to file in qpm-rust v2

### ISSUE THREAD

**Issue by raineaeternal** [QuestPackageManager/QPM.CLI#13]
Would be nice for GH Actions specifically

**Comment by RedBrumbler** [comment]
you can just do this by using `> file.log`

**Comment by Fernthedev** [comment]
I agree, unless you can elaborate further as to why file logging should be integrated in qpm-rust itself I don't see an important reason for it to be implemented. 
Personally I don't understand the use case in Github Actions itself either.

Sounds cool, but I don't fully understand the necessity.
